package.json

```diff
@@ -47,7 +47,7 @@
     "node-forge": "^0.9.1",
     "node-jose": "^1.1.4",
     "nodemon": "^2.0.4",
-    "pg": "^8.3.0",
+    "pg": "^7.8.12",
```

Comment:
I don't think we want to downgrade this. Might require a rebase.

Reply:
The new version was causing problems with deployments on our side so rolled it back. I've now included support for this version which has been tested as working on our deployments.